xcm/xcm_token_locations.json

```diff
@@ -105,40 +311,54 @@
         "interiors": {
           "v1": [
             {
-              "Parachain": 2000
+              "parachain": 2000
             },
             {
-              "GeneralKeyV1": "0x0080"
+              "generalKeyV1": "0x0080"
             }
           ],
           "v3": [
             {
-              "Parachain": 2000
+              "parachain": 2000
             },
             {
-              "GeneralKeyV3": {
+              "generalKeyV3": {
                 "length": 2,
                 "data": "0x0080000000000000000000000000000000000000000000000000000000000000"
               }
             }
           ]
         }
       },
+      {
+        "symbol": "LKSM",
```

Comment:
`LKSM` is a token in Kusama ecosystem. It must have `v3` version for `interiors` block

Reply:
Fixed